### ISSUE TITLE
Dead

### ISSUE THREAD

**Issue by hanoncs** [hMatoba/piexifjs#78]
Should probably mark this repo as dead since its no longer supported by anyone.

**Comment by fatso83** [comment]
Yeah, it says that version 2.0 is being worked on and in the 2.0 branch it says it will be released soon. That was 2 years ago. There are better supported libs for reading exif, but not sure about the writing bit.

**Comment by stackcoder** [comment]
Meanwhile you can write exif data using sharp (based on libvips): https://sharp.pixelplumbing.com/api-output#withmetadata
But there might no other native nodejs implementation.

**Comment by dlong500** [comment]
@stackcoder using sharp is probably the easiest option, but I just confirmed with the author that it is currently impossible to modify exif data without re-encoding the image data which is unfortunate.  So there is still a need for a javascript exif writer that doesn't alter the image data.

**Comment by Oudwins** [comment]
@dlong500 May I ask why it is desired for the image data to be unaltered?

**Comment by dlong500** [comment]
> @dlong500 May I ask why it is desired for the image data to be unaltered?

Because re-encoding the image data is always going to result is some loss of quality (at least for a lossy format).

**Comment by Chew** [comment]
I was hoping to find a package that let me strip EXIF data (specifically location and such). This package _seemed_ good, but no TypeScript typings makes it difficult to use. Additionally, I'm trying to use a Cloudflare Worker, and I'm not sure what would be natively compatible with them as well. If anyone has better suggestions than this library, I'd love to hear it.
